MatrixKit/Views/MXKReceiptSendersContainer.h

```diff
@@ -31,6 +31,17 @@ typedef NS_ENUM(NSInteger, ReadReceiptsAlignment)
     ReadReceiptAlignmentRight = 1,
 };
 
+/*
+ Protocol to provide interface for actions related to the MXKReceiptSendersContainer view
+ */
+@protocol MXKRecieptSendersContainerDelegate <NSObject>
+
+@optional
+
+- (void)didTapReceiptsContainerWithRestClient:(MXRestClient *)restClient RoomMembers:(NSArray *)roomMembers avatars:(NSArray *)avatars recieptDescriptions:(NSArray *)recieptDescriptions;
+
```

Comment:
About a protocol for a delegate, a good practice is to return the concerned instance as the first parameter:
`- (void)receiptSendersContainer:(MXKReceiptSendersContainer*)receiptSendersContainer...`

then the delegate is able to use the instance's property (you don't have to pass them in parameters), like the REST client here.

BTW after reviewing your PRs, I regret to say that we will not be able to use a delegate at the MXKReceiptSendersContainer level. As we can see in the usage of this delegate in Riot ios (see RoomDataSource file) this is not easy to define the actual delegate. You decided to use `[AppDelegate theDelegate].masterTabBarController.currentRoomViewController` as delegate but nothing guarantee that the current data source is working for this view controller (there are several room view controller instances in the app, like the one used to display the context of a room message resulting of a search session...)

To sum up, you should remove the definition and the usage of this new delegate. I will expose to you an other solution based on the `- (void)cell:(id<MXKCellRendering>)cell didRecognizeAction:(NSString*)actionIdentifier userInfo:(NSDictionary *)userInfo;` defined in MXKCellRenderingDelegate protocol. Sorry to have missed that point before.